rpc_jobs/irr_role_test.yml

```diff
@@ -67,6 +67,27 @@
     jobs:
       - 'RPC-IRR_{repo}-{series}-{image}-{context}'
 
+- project:
+    name: "RPC-Leapfrog-Jobs"
+    repo:
+      - rpc-ceph:
+          irr_repo_url: https://github.com/rcbops/rpc-leapfrog
+    series:
+      - undefined
```

Comment:
Does the series need to be set?

Reply:
normally this is set to master but is largely unused, here this will be overridden within the job context and used here: [ https://github.com/rcbops/rpc-upgrades/blob/master/tests/aio-create.sh#L70-L77 ].

Reply:
Cool, lgtm